CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+## Code of Conduct
+
+FluxCD toolkit follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
```suggestion
The Flux project follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Reply:
This was stolen from https://github.com/fluxcd/flux2/blob/main/CODE_OF_CONDUCT.md

Reply:
but sure... fixing :-)